src/Makefile.base

```diff
@@ -17,10 +17,12 @@ ccflags-y += -I$(JERRY_BASE)/jerry-core
 ccflags-y += -I$(ZEPHYR_BASE)/drivers
 ccflags-y += -I$(ZJS_BASE)/outdir/include
 
+ifeq ($(ZJS_DGRAM), y)
 ifeq ($(BOARD), arduino_101)
 obj-y += ../deps/zephyr/samples/bluetooth/gatt/ipss.o
```

Comment:
See whether it works to enable OCF and dgram at the same time, since both include this.